swagger2.0/oic.sec.pstat.swagger.json

```diff
@@ -134,6 +134,15 @@
           "type": "integer",
           "readOnly": true
         },
+        "onboardself": {
+          "description": "Determines whether a DOTS or OBT Device will onboard itself from RFOTM.",
+          "type": "boolean"
+        },
+        "onboardas": {
```

Comment:
We have to remember to remove this based on the recent change to the CR.

Reply:
Updated